sources/main.py

```diff
@@ -21,6 +25,64 @@ async def ping(interaction):
     await interaction.response.send_message('pong')
 
 
+def replace_domain(url: ParseResult):
+    """
+    Replace an original domain by a fixer
+    :param url: an original parsed URL
+    :return: a fixed parsed URL
+    """
+    domains = {
+        'reddit.com': 'rxddit.com',
+        'tiktok.com': 'vxtiktok.com',
+        'vm.tiktok.com': 'vm.vxtiktok.com',
+        'twitter.com': 'vxtwitter.com',
+    }
+    domain = url.netloc
+    for key, value in domains.items():
+        domain = domain.replace(key, value)
```

Comment:
It will cause an issue when the original link contains a fixer URL, i.e. `vxtwitter.com` -> `vxvxtwitter.com`